### PR TITLE
npm package fixes, tested manually.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -134,7 +134,8 @@
     "minimist": "1.2.6",
     "ansi-regex": "5.0.1",
     "async": "2.6.4",
-    "eventsource": "1.1.2"
+    "eventsource": "1.1.2",
+    "terser": "4.8.1"
```
Comment (FuriousLlama) [inline frontend/package.json:138]: this one is new

File: frontend/package.json [hunk 2]
```diff
@@ -19,8 +19,8 @@
     "keycloak-js": "12.0.4",
     "leaflet": "1.7.1",
     "lodash": "4.17.21",
-    "moment": "2.29.2",
-    "moment-timezone": "0.5.33",
+    "moment": "2.29.4",
+    "moment-timezone": "0.5.35",
```
Comment (FuriousLlama) [inline frontend/package.json:23]: dont forget to update the package-lock file as well

**Review by asanchezr — APPROVED** [review]

File: frontend/package.json (hunk 1 above)
Comment (devinleighsmith) [reply frontend/package.json:138]: sort of, it is a transitive dependency.

File: frontend/package.json (hunk 2 above)
Comment (devinleighsmith) [reply frontend/package.json:23]: its there.

**Pull request merged**